test/index.js

```diff
@@ -460,8 +460,63 @@ describe('koa-body', () => {
           done();
         });
     });
+  });
+
+  describe('parsedMethods options',  () => {
+    beforeEach( () => {
+      //push an additional, to test the multi query
+      database.users.push({ name: 'charlike' });
```

Comment:
Wouldn't we have to push in at least two of these to actually test if we are deleting multiple items?

Reply:
Disregard. We reset the "test database" before every test.